lib/ingram_micro/elements/outbound_purchase_order_information.rb

```diff
@@ -6,14 +6,7 @@ class IngramMicro::OutboundPurchaseOrderInformation < IngramMicro::BaseElement
     purchase_order_event: nil,
     currency_code: nil,
     comments: nil
-  }
-
-  def initialize(options = {})
```

Comment:
why was this removed? irc i added this since we didn't need a `customer_id` here and having it there was problematic